src/System/RequirementsManager.php

```diff
@@ -112,6 +112,11 @@ public function getCoreRequirementList(\DBmysql $db = null): RequirementsList
             false,
             __('Enable usage of ChaCha20-Poly1305 encryption required by GLPI. This is provided by libsodium 1.0.12 and newer.')
         );
+        $requirements[] = new Extension(
+            'openssl',
+            false,
```

Comment:
Do you think it could be possible to make this requirement optional? Maybe we could "disable" the OAuth server / API when extension is not loaded, and add a message like "To use XXX, you must first enable the opensssl PHP extension." in related pages.

Reply:
Is it not included with default distributions of PHP?
At least for me, there wasn't a separate extension to install or enable.

Reply:
I do not really know, this point should be checked. Anyway, it can be done in another PR, if we figure out that this extension is not available on default PHP distributions for some OS.